components/director/internal/destinationfetchersvc/client.go

```diff
@@ -237,9 +259,10 @@ func (c *Client) buildRequest(ctx context.Context, url string, page string) (*ht
 
 // FetchDestinationSensitiveData returns sensitive data of a destination
 func (c *Client) FetchDestinationSensitiveData(ctx context.Context, destinationName string) ([]byte, error) {
-	url := fmt.Sprintf("%s%s/%s", c.apiURL, c.apiConfig.EndpointFindDestination, destinationName)
-	log.C(ctx).Infof("Getting destination data from: %s \n", url)
-	req, err := http.NewRequest(http.MethodGet, url, nil)
+	fetchURL := fmt.Sprintf("%s%s/%s", c.apiURL, c.apiConfig.EndpointFindDestination, destinationName)
+	log.C(ctx).Infof("Getting destination data from: %s", fetchURL)
```

Comment:
This log could be removed.
We could add one debug log at the beginning of FetchDestinationsSensitiveData (service.go) where we print the tenant id + all destination names in the request.

Reply:
+The log on 307 `log.C(ctx).Infof("Fetching data for destination: %s \n", destinationName)` seems useless, or at least it should be on debug.